CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 -----
 
+## `v2.2.0-beta`
+
+- Uses go-autorest version v7.0.5.
+- Update packages jwt-go and crypto in go-autorest and SDK.
```

Comment:
this is a bit cryptic. You should probably say `Updated the versions of packages "a" and "b" in glide.lock.`